src/verify.c

```diff
@@ -1024,10 +1024,13 @@ enum swupd_code verify_main(void)
 		}
 	}
 
+	info("Download missing manifests ...\n");
```

Comment:
I know I'm late to this review, but shouldn't this be "Downloading missing manifests..." instead of "Download..."?

Reply:
+1 to change to downloading. @karthikprabhuvinod Can you change that?

Reply:
Agreed.. Will do